bin/clean.js

```diff
@@ -0,0 +1,12 @@
+var rimraf = require('rimraf');
+var fs = require('fs');
+
+var dir = '../build';
+
+rimraf(dir, function(err){
```

Comment:
@andrepcg Could you please adjust it to fit the codestyle used through the codebase? I mean spaces after `if`, after anonymous `function` keyword, etc. etc.?